distributed/queues.py

```diff
@@ -155,6 +155,18 @@ class Queue:
 
        This object is experimental and has known issues in Python 2
 
+    Parameters
+    ----------
+    name: string (optional)
+        Name used by other clients and the scheduler to identify the queue. If
+        not given, a random name will be generated.
+    client: Client (optional)
+        Client used for communication with the scheduler. Defaults to the
+        value of ``_get_global_client()``.
```

Comment:
Maybe instead of specifying a private function here we can just state this in words? What do you think?

Reply:
As a general principle, I agree; documentation is part of the public API and should be agnostic towards private members. I put it this way to make it consistent with the documentation of [`Future`](https://distributed.readthedocs.io/en/latest/api.html#future), so if we change it here, maybe we should take care of that one while we're at it?

I considered just replacing it with an explanation (something like "the default global client"?) but then again, @mrocklin pointing out to me the existence of the relevant function is what helped me realize what was actually happening.

Out of these, `Lock` is a bit special as it uses `self.client = client or _get_global_client() or get_worker().client`.

Reply:
Fair enough.